### ISSUE TITLE
add search filters

### ISSUE THREAD

**Issue by abdulhannanali** [abdulhannanali/the-peace-pledge#17]
search filters can be used in order to provide user with more useful information. Some of the useful filters I can implement are 
- Search By Country
- Search By Name

**Comment by sudheesh001** [comment]
:+1: Agree !

**Comment by abdulhannanali** [comment]
Yeah I am going to continue this effort for the peace pledge. I think I can get some signups from the beautiful internet
